src/meshes/unstructured_mesh.jl

```diff
@@ -3,271 +3,308 @@
 # we need to opt-in explicitly.
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
-#! format: noindent
+    #! format: noindent
 
-"""
-    UnstructuredMesh2D <: AbstractMesh{2}
+    """
+        UnstructuredMesh2D <: AbstractMesh{2}
 
-An unstructured (possibly curved) quadrilateral mesh.
+    An unstructured (possibly curved) quadrilateral mesh.
 
-    UnstructuredMesh2D(filename; RealT=Float64, periodicity=false)
+        UnstructuredMesh2D(filename; RealT=Float64, periodicity=false)
 
-All mesh information, neighbour coupling, and boundary curve information is read in
-from a mesh file `filename`.
-"""
-mutable struct UnstructuredMesh2D{RealT <: Real,
-                                  CurvedSurfaceT <: CurvedSurface{RealT}} <:
-               AbstractMesh{2}
-    filename              :: String
-    n_corners             :: Int
-    n_surfaces            :: Int # total number of surfaces
-    n_interfaces          :: Int # number of interior surfaces
-    n_boundaries          :: Int # number of surfaces on the physical boundary
-    n_elements            :: Int
-    polydeg               :: Int
-    corners               :: Array{RealT, 2}  # [ndims, n_corners]
-    neighbour_information :: Array{Int, 2}  # [neighbour node/element/edge ids, n_surfaces]
-    boundary_names        :: Array{Symbol, 2} # [local sides, n_elements]
-    periodicity           :: Bool
-    element_node_ids      :: Array{Int, 2} # [node ids, n_elements]
-    element_is_curved     :: Vector{Bool}
-    surface_curves        :: Array{CurvedSurfaceT, 2} # [local sides, n_elements]
-    current_filename      :: String
-    unsaved_changes       :: Bool # if true, the mesh will be saved for plotting
```

Comment:
This alignment was a cool feature in JuliaFormatter.jl.

Reply:
Does it really increase the readability of the code though? (This) code isn't read column by column but row by row.

While creating Runic I have thought a bit about things like this. I also used to sometimes align things like thisbut I have realized that usually it is just to make me "feel better", it doesn't really increase the readability. When you stop worrying about things like this while writing the code (e.g. press space multiple times to align) and just accept the formatters rules it kind of gives a similar peace of mind. :slightly_smiling_face:

Reply:
I think it does. I can clearly see which are scalar values and which are the big arrays that I might be looking for.